package.json

```diff
@@ -36,7 +36,7 @@
     "glob-expand": "^0.2.1"
   },
   "peerDependencies": {
-    "typescript": "^2.0.6 || >=2.1.0-dev || >=2.2.0-dev"
+    "typescript": "^2.0.6 || ^2.1.5 || >=2.2.0-dev"
```

Comment:
`^2.0.6` contains `2.1.5`. this change is not required. 😉 

https://runkit.com/npm/semver

```
var semver = require('semver');

console.log(semver.satisfies('2.0.10', '^2.0.6 || >=2.1.0-dev || >=2.2.0-dev'));
console.log(semver.satisfies('2.1.5', '^2.0.6 || >=2.1.0-dev || >=2.2.0-dev'));
console.log(semver.satisfies('2.1.0-dev.20161110', '^2.0.6 || >=2.1.0-dev || >=2.2.0-dev'));
console.log(semver.satisfies('2.2.0-dev.20170124', '^2.0.6 || >=2.1.0-dev || >=2.2.0-dev'));
```